stylesheets/resetPassword.css

```diff
@@ -0,0 +1,109 @@
+@import url(http://fonts.googleapis.com/css?family=Source+Sans+Pro:400,200,300,700,900&subset=latin,latin-ext);
```

Comment:
This file definitely doesn't need all those different font weights (400, 200, 300, 700). The usage below suggest that you're only using weight 400 so you should be able to get rid of the rest (fast loading time)